src/portalbackend/PortalBackend.Migrations/Seeder/Data/identities.json

```
@@ -120,5 +120,45 @@
     "user_status_id": 1,
     "user_entity_id": "e8bc6470-28ee-4c40-a2d9-27c6e78f303b",
     "identity_type_id": 2
+  },
+  {
+    "id": "48fb21ad-4439-455b-92bb-8c54801e3eab",
+    "company_id": "2dc4249f-b5ca-4d42-bef1-7a7a950a4f87",
+    "date_created": "2024-09-27 10:01:33.439000 +00:00",
+    "user_status_id": 1,
+    "user_entity_id": "obsolete-obso-lete-obso-obsolete0001",
+    "identity_type_id": 2
```

UserEntityId is nullable:
```suggestion
    "user_status_id": 1,
    "user_entity_id": null,
    "identity_type_id": 2
```